operations/finance/spending-company-money/how-to-use-accrualify/how-to-submit-a-po.md

```diff
@@ -0,0 +1,145 @@
+# Submitting a Purchase Order
+
+Log into Accrualify and go to the Purchase Module on the left hand side of the screen. Select the orange **Request PO** button.
+
+![](../../.gitbook/assets/How-to-submit-a-PO-1.PNG)
+
+A **Request a New Purchase Order** form will appear prompting you to fill out the primary information. The screenshot below highlights all the fields needed to be populated to successfully complete a submission.
+
+![](../../.gitbook/assets/How-to-submit-a-PO-2.PNG)
+
+Mapping key to each field and purpose:
+
+1. **Vendor/Supplier:** the field is not a drop down list but a search bar. The vendor name must match exactly for the vendor to appear available. Refer to Finding a Vendor on the steps to successfully locate a vendor.
+      * **Add Vendor:** Only click this button if you have taken all previous steps to determine if your vendor exists as an active vendor in our repository. Refer to section Requesting a Vendor.
+2. **Payment terms:** Mattermost standard payment terms is Net 30. If your vendor accelerated payment terms like Net 15, Net 7, or due on receipt,  this will require additional approval from Finance. You should reach out to procurement@mattermost.com or Finance Team Channel
+3. **Amount:** This field is intentionally grayed out as it will pull from the section PO Expenses further down screen window.
+4. **Start Date/End Date:** These are the dates for when the services will start and complete. For goods being delivered on a specific date, the start and end date should be the same.
+5. **Date/Receive By Date:** This is automatically populated based on today’s date. This may be changed, but is not recommended as this will account at the request date. The **Receive by** field is the date you would ideally like to PO to be approved. Note that we still maintain standard SLAs. If the request is urgent, we will do our best to accommodate but make no guarantees. If you are working with a new vendor on negotiations, it is advisable to bring the Procurement Team as early as possible to avoid any potential delays or missed dates.
```

Comment:
```suggestion
5. **Date/Receive By Date:** This is automatically populated based on today’s date. This may be changed, but is not recommended as this will account at the request date. The **Receive by** field is the date you would ideally like to PO to be approved. Note that we still maintain standard SLAs. If the request is urgent, we will do our best to accommodate but make no guarantees. If you're working with a new vendor on negotiations, it's advisable to bring the Procurement team as early as possible to avoid any potential delays or missed dates.
```